images/kubekins-e2e-v2/Dockerfile

```diff
@@ -74,6 +74,13 @@ RUN wget -O google-cloud-sdk.tar.gz -q $GCLOUD_SDK_URL && \
     gcloud components install alpha beta && \
     gcloud info | tee /workspace/gcloud-info.txt
 
+# Install aws cli
```

Comment:
I'm concerned about making this image even more risky to update.

Where are we using the CLI instead of some tool that links the SDK?

Reply:
(if we could remove gcloud we should, but that requires eliminating kube-up realistically)

Reply:
az cli is installed via apt and isn't an issue. You should be able to tweak it further on job execution if necessary.

The aws cli is also like that as well, the zip gets dropped in /usr/local/bin along with the self contained python libraries.

They can be updated without affecting the rest of the image tooling.

Reply:
>  So many capz/sig-windows jobs install the cli on every run :( hundreds of times a day
>
> The aws cli is also like that as well, the zip gets dropped in /usr/local/bin along with the self contained python libraries.

Yes, but on the other hand this means the version installed is controlled by the test code.
We have lots of things we live-install and increasingly only depend on the image for bootstrapping (e.g. go 1.21+ toolchain)